lend/src/components/Referral/index.tsx

```diff
@@ -45,7 +45,7 @@ const RbnButtonWrapper = styled.div`
 const CopyToClipboardButton = styled(Button)`
   background: ${colors.buttons.secondaryBackground};
   color: ${colors.buttons.secondaryText};
-  width: 74%;
+  width: 78.75%;
```

Comment:
can we set the width % on the parent container?

Reply:
The parent container `RbnButtonWrapper` contains `CopyToClipboardButton` and `TwitterButton`, which is why I set the width for each child in each of the two child components respectively